### PR TITLE
Add Kube README.md

### PR DESCRIPTION
**Pull request by rgrandl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,239 @@
...
+// Install the GKE authentication plugin.
+$ gcloud components install gke-gcloud-auth-plugin
+
+// Initialize the local environment.
+$ gcloud init
+
+// Enable google APIs.
+$ gcloud services enable container.googleapis.com
+
+// Example to create a cluster.
```
[the remark this answers is not in the document]
Comment (spetrovic77) [reply README.md:51]: I would use concrete values for name zone etc. How about:

```
name = example
zone = us-west1-a
x = 3
type = e2
```

People should be smart enough to change those values if needed.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,239 @@
...
+
+```console
+// Make sure your AWS credentials are correctly generated.
+$ aws configure
+
+// Check ~/.aws directory to see if AWS was configured successfully.
+// You should have a /config and a /credentials file.
+$ ls ~/.aws
+
+// If you want to set a different region as default for your cluster.
```
Comment (spetrovic77) [inline README.md:68]: I would delete this line (too much)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,239 @@
...
+$ aws eks update-kubeconfig --region [REGION_NAME]
+
+// Make sure the metrics server is started. One way to do this is to run
+// the following command.
+$ kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
+
+// Example to create a cluster.
+$ eksctl create cluster --name <name> --region <region> --node-type=<type> --nodes=<x>
```
Comment (spetrovic77) [inline README.md:76]: see above: just pick some values here.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,239 @@
...
+```
+
+After these steps, if you run `kubectl get nodes`, you should see a Kubernetes
+cluster up and running.
+
+### 3. Kube deployer
```
Comment (spetrovic77) [inline README.md:82]: Too many "kube"s. 😄 

Can you rewrite to: `Install the weaver-kube command`

**Review by spetrovic77 — APPROVED** [review]
Looks great Robert. At some point, much of this stuff should be moved to the website. But it's fine here for now.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,239 @@
-# weaver-k8s
-Run Service Weaver applications on vanilla Kubernetes.
+# weaver-kube
+
+This repository contains Go libraries for deploying Service Weaver applications
+in any [Kubernetes][kubernetes] environment, i.e. on [GKE][gke], [EKS][eks],
+[AKS][aks], [on-prem][kvm], etc.
```
[the remark this answers is not in the document]
Comment (rgrandl) [reply README.md:5]: Done

File: README.md [hunk 6]
```diff
@@ -1,2 +1,239 @@
...
+
+Visit [https://serviceweaver.dev][website] to learn about Service Weaver.
+
+## Overview
+
+![](internal/docs/kube_architecture.png)
+
+The `kube` deployer builds a [Docker][docker] image for your Service Weaver
+application, and generates all the configurations (e.g., Kubernetes [pods][pods],
+[replica sets][rs], [deployments][deployment], [services][service],
```
[the remark this answers is not in the document]
Comment (rgrandl) [reply README.md:15]: Done

**Review by rgrandl — COMMENTED** [review]
Thanks for the thorough review.

**Pull request merged**